templates/kubernetes/onpremises/delete-playbook.yaml.tpl

```diff
@@ -19,8 +19,8 @@
   tags:
     - reset-k8s
 
-- name: Reset etcd master nodes
-  hosts: master
+- name: Reset etcd nodes
+  hosts: etcd
```

Comment:
Should this be done on master,etcd and maybe limiting the stop etcd only on the etcd nodes? Everything else can be done safely to all the nodes, and etc/kubernetes/pki must be cleaned up also on the masters